src/App.js

```diff
@@ -1,25 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends Component {
+  state = {
+    date: null,
```

Comment:
good job 👍  but why do you set 'null' for the date?
It causes that the clock appears one second later after the page is downloaded.